#include "team_server_player.h"
#include "src/game_logic/thread_local/game_registry.h"
#include "src/network/message_system.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

///<<<rpc begin
void ServerPlayerTeamServiceImpl::EnterScene(entt::entity player,
    const ::TeamTestRequest* request,
    ::TeamTestResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

///<<<rpc end
